platform/app/public/config/e2e.js

```diff
@@ -53,7 +53,7 @@ window.config = {
         supportsFuzzyMatching: false,
         supportsWildcard: true,
         staticWado: true,
-        singlepart: 'bulkdata,video,pdf',
+        singlepart: 'video',
```

Comment:
should we change other servers as well? like dcm4chee one?

Reply:
hello @sedghi please do am experiencing the same issue (no online PDF viewer installed) with orthanc .